modules/oe_content_timeline_field/tests/Kernel/TimelineFieldTest.php

```diff
@@ -135,6 +135,12 @@ public function testTimeline(): void {
           'title' => 'Item 3',
           'body' => 'Item 3 body',
         ],
+        [
+          'label' => 'Test € — ☺',
+          'title' => 'Item 4',
+          'body' => 'Item 4 body',
+          'format' => 'format_☺',
```

Comment:
If we don't change the type for `format` then this will fail, so either we change it or we amend this, same below